hxinsight-extension/src/main/resources/alfresco/module/alfresco-hxinsight-connector-hxinsight-extension/context/service-context.xml

```diff
@@ -158,7 +158,43 @@
         <constructor-arg name="permissionService" ref="PermissionService"/>
     </bean>
 
-    <bean class="org.alfresco.hxi_connector.hxi_extension.rest.api.QuestionsEntityResource" />
+    <bean id="questionService" class="org.alfresco.hxi_connector.hxi_extension.service.QuestionServiceImpl">
+        <constructor-arg name="hxInsightClient" ref="hxInsightClient"/>
+        <constructor-arg name="authenticationService" ref="AuthenticationService"/>
+        <constructor-arg name="personService" ref="PersonService"/>
+    </bean>
+
+    <bean id="QuestionService" class="org.springframework.aop.framework.ProxyFactoryBean">
```

Comment:
I'm not sure if we need all these wrappers🤔

Reply:
I agree it's not necessary - `AuthenticationService` and `PersonService` has already configured security, transactions, etc. - wrappings removed